Language/Functions/Interrupts/interrupts.adoc

```diff
@@ -17,7 +17,7 @@ subCategories: [ "Interrupts" ]
 
 [float]
 === Description
-Re-enables interrupts (after they've been disabled by link:../nointerrupts[noInterrupts()]. Interrupts allow certain important tasks to happen in the background and are enabled by default. Some functions will not work while interrupts are disabled, and incoming communication may be ignored. Interrupts can slightly disrupt the timing of code, however, and may be disabled for particularly critical sections of code.
+Re-enables interrupts (after they've been disabled by link:../nointerrupts[noInterrupts()]. Interrupts allow certain important tasks to happen in the background and are enabled by default. Some functions will not work while interrupts are disabled, and incoming communication may be ignored. Interrupts can slightly disrupt the timing of code, however, and may be disabled for particularly critical sections of code. They can also cause races, code that behaves differently, and incorrectly, in response to tiny changes in timing. See `interrupts()` for further information.
```

Comment:
Hi @MalcolmBoura. Thanks for taking the time to submit a pull request!
> See `interrupts()` for further information.

I don't understand this instruction to the reader. What do you mean by "see `interrupts()`"?

Reply:
Oops, sorry. Was meant to be a link to the attachInterrupt() page and the surrounding text could be clearer. I suggest:

See the link:../attachInterrupt[attachInterrupt()] page for further information.

I will see if I can add that to the branch but I have not used git or github much.

 I have proposed an update to the attachInterrupt() page which explains the problem and how to fix it.

Reply:
I think this version of the link is more likely to work. I will read the documentation again as soon as I can.

See the link:../External%20Interrupts/attachInterrupt[attachInterrupt()] page for further information.